### DISCUSSION TITLE
Textlayer and Annotations out of alignment

### DISCUSSION THREAD

**Discussion by GarikHk** [wojtekmaj/react-pdf#1546] [Q&A]
https://github.com/wojtekmaj/react-pdf/issues/332

Hey, I am having issues similar to this [Textlayer out of alignment](https://github.com/wojtekmaj/react-pdf/issues/332) one. On resize, when I make my window smaller the textLayer gets out of alignment with what's rendered on the canvas. I was thinking maybe the font itself doesn't change its size and only the container does. I have tried most of the suggested solutions there but it doesn't seem to work. Any help would be nice...

`
.document-content .react-pdf__Page canvas, 
.react-pdf__Page__textContent, 
.react-pdf__Page__annotations { 
  max-width: 100%;
  height: auto !important; 
}
`
I added this following porvided samples in this repo so the page canvas 
could be responsive and textLayer/annotations wouldn't get out of their container.

![pdfResizeBug](https://github.com/wojtekmaj/react-pdf/assets/118485559/a2d985d6-289f-4255-9a3e-bb082de77f11)

**Comment by sohaibaumer** [comment]
Use the react-pdf-scale package to scale the text layer independently of the canvas. You could do something like this:

![image](https://github.com/wojtekmaj/react-pdf/assets/120295412/c4d9795f-272f-42c2-aa15-8913e410bed6)

**Reply by wojtekmaj** [reply]
There's no such package: https://www.npmjs.com/package/react-pdf-scale

**Comment by wojtekmaj** [comment]
Please refer to https://github.com/wojtekmaj/react-pdf/issues/332#issuecomment-1632119019.
